pages/auth.vue

```diff
@@ -1,61 +1,4 @@
 <script lang="ts" setup>
+    import { tradeCodeForToken } from "~/assets/ts/auth";
     tradeCodeForToken();
```

Comment:
Der name `tradeCodeForToken` würde eher so etwas suggestieren, was ich auch besser finden würde
```suggestion
    const params = ...
    if(params.error) ...
    const code = params.get('code')
    const token = tradeCodeForToken(code);
    ...
```